composer/algorithms/progressive_resizing/README.md

```diff
@@ -55,7 +55,8 @@ TODO(Cory): A brief description of how this works under the hood.-->
 
 ## Suggested Hyperparameters
 
-We found `initial_scale = 0.5` (starting training on images where each side length has been reduced by 50%) and `finetune_fraction = 0.2` (reserving the final 20% of training for full-sized images) to work well for ResNet-50 on ImageNet.
+We found `initial_scale = 0.5` (starting training on images where each side length has been reduced by 50%), `finetune_fraction = 0.2` (reserving the final 20% of training for full-sized images), 
+`delay_fraction = 0.5` (reserving the first 50% of training for initial\_scale-sized images) and `size_increment = 4` (pegging image size to the nearest multiple of 4) to work well for ResNet-50 on ImageNet.
 These are the hyperparameters we used for our runs in the [MosaicML Explorer](https://app.mosaicml.com/explorer/imagenet).
```

Comment:
```suggestion
```

We might want to remove the line until we do another data release in Explorer, as the suggested hparams will no longer match those used to generate that data.

Reply:
Cool. I thought this statement would be true eventually so I kept it. I'll go ahead and remove it.